hapi-proto/src/main/proto/ScheduleGetInfo.proto

```diff
@@ -47,6 +54,9 @@ message ScheduleInfo {
   Timestamp expirationTime = 8; // The epoch second at which the schedule will expire
 }
 
+/*
+Response when the client sends the node ScheduleGetInfoQuery
```

Comment:
I feel like this one sounds a bit confusing.
Maybe: "Response that gets returned, when the client sends the node a ScheduleGetInfoQuery"